charts/zot/Chart.yaml

```diff
@@ -3,4 +3,4 @@ appVersion: v2.0.3
 description: A Helm chart for Kubernetes
 name: zot
 type: application
-version: 0.1.53
+version: 0.1.54
```

Comment:
Please rebase and increment again. We just made a zot release and the new app version was included in .54